### ISSUE TITLE
Set default view for types

### ISSUE THREAD

**Issue by josh-65** [files-community/Files#5564]
**Is your feature request related to a problem?**
Currently for every type of file (e.g. photos) I have to set my layout mode manually. There should be options in the settings to change the layout mode for folder types like file explorer can.

**Describe the solution**
In settings the can be a new category called default view or something like that. Then it shows all the types of folder that windows has (e.g. Documents) with a drop down next to it showing the view mode (Detail, Tile, Grid (small, medium and large)) it will set to.

**Scope**
| Capability | Priority |
| :---------- | :------- |
| Allow default view for folder types | Must |
| Add an option in folder properties to change the folder type | Should |

**Comment by gave92** [comment]
I think it's supposed to do something like this automatically, right @d2dyno1? Not on folder type but if a folder is full of pictures it should switch to gridview.

**Comment by josh-65** [comment]
Properly but it doesn't let me choose what style I want it in. Photos will always go to small grid mode when I could what it to go to tile view. Music will go to detail view when I would want it in small grid mode.

**Comment by d2dyno1** [comment]
Yeah, It'll more often change to gridview. Tiles have been completely removed from Adaptive Layout predictions. Also, you can override Adaptive Layout and Files will remember your choice.

**Comment by gave92** [comment]
It will remember the choice for just that folder, but won't change the predicted layout mode for all other picture folders, right?

**Comment by d2dyno1** [comment]
Yes, it'll stay overridden in that one folder only.

**Comment by yaichenbaum** [comment]
File Explorer has an option in the properties window that lets you select what kind of content is stored in the folder. Using this option will automatically set the layout and columns to match the option you selected.

I'm not sure if we want to put it in the properties window but we'll want to implement something like this.

**Comment by yaira2** [comment]
Do you still think this is necessary? We allow setting the default layout mode and you can override individual folders as needed.

**Comment by Josh-65** [comment]
Would be nice, since doing it individually for many sub folder is annoying. Maybe an option to apply current folder layout to all subfolders would work in the right click menu.

**Comment by Cinistello** [comment]
I have a situation where adaptive layout isn’t working great, because I have many folders of other folders that are given the list view, and then when I do get down to images, half of them will be clip format (similar to psd) and it will be a list. But the ones with more pngs that the adaptive layout does notice work great. I'm manually changing them but its a lot due to how I organize my art over many years. I have one "Images" folder at the top that I wish I could set a layout to and just apply it to allll those subfolders

**Comment by yair100** [comment]
@Cinistello which file extension? We can look into using grid layout for that file type.
